roscala/src/test/scala/coop/rchain/rosette/TransitionSpec.scala

```diff
@@ -1,10 +1,12 @@
 package coop.rchain.rosette
 
+import coop.rchain.rosette
 import coop.rchain.rosette.Meta.StdMeta
 import coop.rchain.rosette.expr.{LetExpr, TupleExpr}
 import org.scalatest._
 
 class TransitionSpec extends FlatSpec with Matchers {
+  val t = Tuple.apply(Ob.NIV)
```

Comment:
Did you forget to remove this? I don't see it being used anywhere.

Reply:
Yep, thanks!